pytorch/buildspec.yml

```diff
@@ -90,68 +90,68 @@ images:
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION,
-  #                "-ec2-example" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /example, /Dockerfile., *DEVICE_TYPE ]
+  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-sagemaker" ]
+  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile.,
+  #                        *DEVICE_TYPE ]
+  #   target: sagemaker
   #   context:
   #     <<: *TRAINING_CONTEXT
-  # BuildEC2CPUPTInferencePy3DockerImage:
-  #   <<: *INFERENCE_REPOSITORY
-  #   build: &PYTORCH_CPU_INFERENCE_PY3 false
-  #   # enable_test_promotion: false
-  #   image_size_baseline: 4899
-  #   device_type: &DEVICE_TYPE cpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-e3" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
-  #   target: ec2
-  #   context:
-  #     <<: *INFERENCE_CONTEXT
-  # BuildEC2GPUPTInferencePy3DockerImage:
-  #   <<: *INFERENCE_REPOSITORY
-  #   build: &PYTORCH_GPU_INFERENCE_PY3 false
+  # BuildPyTorchExampleGPUTrainPy3DockerImage:
+  #   <<: *TRAINING_REPOSITORY
+  #   build: &PYTORCH_GPU_TRAINING_PY3 false
   #   # enable_test_promotion: false
   #   image_size_baseline: 14000
+  #   #base_image_name: BuildEC2GPUPTTrainPy3DockerImage
   #   device_type: &DEVICE_TYPE gpu
   #   python_version: &DOCKER_PYTHON_VERSION py3
   #   tag_python_version: &TAG_PYTHON_VERSION py38
   #   cuda_version: &CUDA_VERSION cu116
   #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-ec2" ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile.,
-  #                        *DEVICE_TYPE ]
-  #   target: ec2
+  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION,
+  #                "-ec2-example" ]
+  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /example, /Dockerfile., *DEVICE_TYPE ]
   #   context:
-  #     <<: *INFERENCE_CONTEXT
+  #     <<: *TRAINING_CONTEXT
+  BuildEC2CPUPTInferencePy3DockerImage:
+    <<: *INFERENCE_REPOSITORY
+    build: &PYTORCH_CPU_INFERENCE_PY3 false
+    # enable_test_promotion: false
+    image_size_baseline: 4899
+    device_type: &DEVICE_TYPE cpu
+    python_version: &DOCKER_PYTHON_VERSION py3
+    tag_python_version: &TAG_PYTHON_VERSION py38
+    os_version: &OS_VERSION ubuntu20.04
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-e3" ]
```

Comment:
this should be "ec2" and not "e3" - please ensure that no other occurrences of "ec2" were changed back to "e3"